### ISSUE TITLE
Allow the turn rotor option to send a UDP command to PSTRotator

### ISSUE THREAD

**Issue by ny4i** [n4af/TR4W#732]
I use PSTRotator rather than a direct connection fromTR4W to the rotor controller. Allow an option to send the UDP command to PSTRotator to turn the rotor instead of trying to use the direct DCU-1 command. This requires a new UDP port and socket option as well as the external rotor control type in case other programs like PSTRotator want could be the target of this command. 

Also make sure this command is not dependent upon a rotator serial port being connected.

It may be possible for PSTRotator to read the UDP rotor command so check that as a possibility.

**Comment by ny4i** [comment]
PSTRotator does have a tracking interface for TR4W although I have not figured out how it does this yet so this may not be needed.

**Comment by ny4i** [comment]
TR4W should add an option called PSTRotator UDP Port AND PSTRotator IP ADDRESS

Default PSTROTATOR UDP PORT to 12000 and PSTRotator IP ADDRESS to 127.0.0.1
There should also be a PSTROTATOR ENABLED = FALSE (default to false)

Add a new ROTATORTTYPE of PSTRotator. THere is already a port of TCP/IP.

Here is the protocol.

UDP Interface PstRotatorAz can receive commands from other programs and report the antenna position via UDP messages. 

For this: • 
Go to "Communication/UDP Control Port..." and set the UDP port number where the program listens.
 The default value is 12000. • The port used for antenna position report is UDP Port + 1. For example, if you set the default UDP Port = 12000, the program reports the antenna position to UDP port = 12001 • Enable UDP Control in Setup
 
PstRotatorAz can receive commands like these: 
<PST><PARK>1</PARK></PST> - stop rotor, switch to manual mode, park antenna 
<PST><TRACK>1</TRACK></PST> - switch to tracking mode 
<PST><TRACK>0</TRACK></PST> - switch to manual mode
 <PST><STOP>1</STOP></PST> - stop rotor, switch to manual mode 
<PST><ON>0</ON></PST> - stop to send commands to rotor (the ON/OFF button will be OFF)
 <PST><ON>1</ON></PST> - start to send commands to rotor (the ON/OFF button will be ON)
 <PST><AZIMUTH>85</AZIMUTH></PST> - set azimuth to 85 deg 
<PST><QRA>KN34BJ</QRA></PST> - send to program the locator (4/6/8 digits) and set the azimuth 
<PST><ANT>1</ANT></PST> - select ANT1 
<PST><ANT>2</ANT></PST> - select ANT2 
<PST><STF>2824500</STF></PST> - tune a SteppIR antenna on 28.245MHz 
<PST><STR>1</STR></PST> - select OmniRig (1/2) used by a SteppIR antenna) 
<PST>AZ?</PST> - in answer to this command the program will report the antenna position to UDP Port + 1 PstRotatorAz reports the antenna position like this: AZ:xxx<CR> 
<PST>TGA?</PST> - in answer to this command the program will report the target azimuth to UDP Port + 1 
PstRotatorAz reports the target azimuth like this: TGA:xxx<CR> Any message must start with <PST> and end with </PST> You can send many commands in one message:
 <PST><STOP>1</STOP><TRACK>0</TRACK><AZIMUTH>85</AZIMUTH></PST>

**Comment by ny4i** [comment]
## Triage & implementation plan

### How rotator control works today
All "point the antenna" actions flow through a single choke point, `RotorControl(Heading)` in `LOGSTUFF.PAS` (called from the Ctrl-P paths in `MainUnit.pas`). It already handles two mechanisms in parallel:
- the **N1MM-format UDP broadcast** (`SendUDPRotorCommand`, gated by `UDPBroadcastRotor`, sent to `UDPBroadcastAddress:UDPBroadcastRotorPort` — default `127.0.0.1:12040`), and
- a **serial rotator** selected by `ActiveRotatorType` (`DCU1/Orion/Yaesu/AlfaSpid`) written to `ActiveRotatorPort`.

A shared Indy UDP client (`udp`) is already created and used for the N1MM send, so PSTRotator can reuse it. `ROTATOR TYPE` is a list-bound config command, so a new enum value appears in the dropdown automatically.

### Design decision
PSTRotator is added as a **new `ROTATOR TYPE`** with its own native UDP protocol (`<PST><AZIMUTH>nnn</AZIMUTH></PST>`). The existing **N1MM broadcast is left completely untouched.** A separate `PSTROTATOR ENABLED` flag was considered and **dropped as redundant** — selecting `ROTATOR TYPE = PSTRotator` is the enable; `ROTATOR TYPE = NONE` is the disable, consistent with every other rotator type.

### Plan
1. **`LOGWIND.PAS`** — add `PSTRotator` to the `RotatorType` enum and `'PSTROTATOR'` to `RotatorTypeSA`.
2. **`LOGSTUFF.PAS`** — new globals `PSTRotatorIPAddress: string[255] = '127.0.0.1'` and `PSTRotatorUDPPort: integer = 12000`; new `SendPSTRotorCommand(Heading)` (modeled on `SendUDPRotorCommand`) that sends `<PST><AZIMUTH>nnn</AZIMUTH></PST>` via the shared `udp` client.
3. **`LOGSTUFF.PAS / RotorControl`** — route `ActiveRotatorType = PSTRotator` to `SendPSTRotorCommand` before the serial gate; N1MM and serial paths unchanged.
4. **`uCFG.pas`** — two new config commands `PSTROTATOR IP ADDRESS` (ctString, default `127.0.0.1`) and `PSTROTATOR UDP PORT` (ctInteger, default `12000`); `CommandsArraySize` +2.

### Scope
This pass implements the **SET** direction only (TR4W → PSTRotator azimuth). PSTRotator's *position readback* on UDP port+1 (`<PST>AZ?</PST>` → `AZ:xxx`) — i.e. PSTRotator-driven tracking back into TR4W — is left as a future enhancement (per the opening comment, that interface isn't yet characterized).

### Related (separate issue, not done here)
The `PortType` label `'TCP/IP'` is transport-specific (the enum value is internally `NETWORK`) and could read `IP`/`NETWORK`. PSTRotator does not use `PortType`, so it's out of scope here. Relabeling it is back-compat-sensitive (the same literal is matched when parsing existing `tr4w.ini`/`.cfg` port values), so it warrants its own issue with backward-compatible parsing.
